### ISSUE TITLE
Add a "cancel queue" button in the "rendering video" screen

### ISSUE THREAD

**Issue by Abwasserrohr** [ReplayMod/ReplayMod#566]
Hello there,

first of all, thank you for making this amazing mod, I really appreciate all the hard work going into it.

From time to time, I mess up some settings in my client and forget about them until I hit "render all" and then started rendering like 10 different keyframes once. Now, if I wanted to exit the rendering, I would have to click on "cancel recording" for each of the rendering processes and wait until each of these are loading, which can take like 5 minutes just to get back.

If there is some kind of button to cancel the whole queue, it would be really useful for people who started a lot of keyframes to render and then wanted to cancel all of them using one button.

**Comment by TheCoderProYT** [comment]
can't you just close minecraft?

**Comment by Abwasserrohr** [comment]
Yeah, I know. But since there is already a cancel recording button, there
could also bei one for cancelling all records in queue. 😅
